statsmodels/robust/tests/test_rlm.py

```diff
@@ -112,11 +112,7 @@ def setup_class(cls):
         cls.res1.h2 = h2
         cls.res1.h3 = h3
 
-
     def setup(self):
-#        r.library('MASS')
-#        self.res2 = RModel(self.data.endog, self.data.exog,
-#                        r.rlm, psi="psi.huber")
```

Comment:
those are part of the documentation for the test reference numbers

Reply:
Great, let's talk about how to turn this commented-out code into useful documentation for the next reader.  For starters: what is `r` and what is `RModel`?

Reply:
obviously it's difficult to find if it has been deleted including the directory
https://github.com/statsmodels/statsmodels/blob/maintenance/0.5.x/statsmodels/tests/rmodelwrap.py
(I'm not into robust right now, so we can talk about this another time. github search finds RModel also in glm, discrete and regression unit tests)

Reply:
The important pieces of information is that it uses the MASS library and the call is something like
`rlm(exog, endog, psi=psi.huber)`
with some corrections to find the appropriate R calling syntax.
There might be a second type of information in what attributes or methods were used of the object returned by the R function call in checking various of our results attributes.

i.e. the full translation of this would be a few lines of valid R code that reproduces the results.

Note, that this can be quite a bit of work, and it's easier to leave it to when we actually need it and are in the neighborhood already.

Reply:
I’ll circle back to this after things quiet down.  Closing.